lib/databaseMethods.js

```diff
@@ -19,7 +19,24 @@ async function getQuestions() {
   return await cursor.toArray();
 }
 
+async function getResults() {
+  const cursor = collection("results").find();
+  return await cursor.toArray();
+}
+async function getResultsByQuestionID(id, answer) {
+  const cursor = collection("results").find({ id: id }, { answer: answer });
+  return await cursor.toArray();
```

Comment:
You might need to create a new id if you want to get a result by the mongoDb id. Also, you just need to findOne, because the id only exists one time.
```suggestion
async function getResultsByQuestionID(id) {
  const objectId = new ObjectID.createFromHexString(id);
  return await collection("results").findOne({ id: objectId });
}
 

  
  
  
  
```